module.txt

```diff
@@ -5,6 +5,7 @@
     "displayName": "Behaviors",
     "description": "A set of behavior traits which could be applied to creatures and NPCs",
     "dependencies": [
+        { "id": "CoreWorlds", "minVersion": "1.1.0", "because": "tests/ClusterDebugger"},
```

Comment:
If only used in tests, this can be an optional (build-time) dependency

```suggestion
        { "id": "CoreWorlds", "minVersion": "1.1.0", "optional": true, "because": "tests/ClusterDebugger"},
```